book/src/SUMMARY.md

```diff
@@ -21,7 +21,6 @@
     - [Opaque types (impl Trait)](./clauses/opaque_types.md)
     - [Well known traits](./clauses/well_known_traits.md)
     - [Well-formedness checking](./clauses/wf.md)
-    - [Coherence](./coherence.md)
```

Comment:
Why remove this?

Reply:
Because the page doesn't exist; it's auto-filled by mdBook to this:

![image](https://user-images.githubusercontent.com/37223377/96947990-6f03bd80-1499-11eb-9002-50373ec58546.png)

See rust-lang/mdbook#1364.

Reply:
Ah oops. It should be `./clauses/coherence.md`. That's an error from #623

Reply:
Fixed.